src/plugins/workspace/public/utils.ts

```diff
@@ -70,6 +70,21 @@ export const featureMatchesConfig = (featureConfigs: string[]) => ({
   return matched;
 };
 
+/**
+ * Calculate all features quantity and selected features quantity in a workspace
+ */
+export const getSelectedFeatureQuantities = (
+  featuresConfig: string[],
+  configurableApps: PublicAppInfo[]
+) => {
+  const featureFilter = featureMatchesConfig(featuresConfig);
+  const selectedApplications = configurableApps.filter((app) => featureFilter(app));
+  return {
+    total: configurableApps.length,
```

Comment:
The `DEFAULT_SELECTED_FEATURES_IDS` is not exists in configurable apps. Shall we add them to the total and selected list?

Reply:
No. Here we need to make features consistent with create page.

Reply:
I think the featureConfig will includes these two feature ids, which means it may return `{total: 0, selected: 2}`. Do we need to filter out these two feature ids?

Reply:
Could you please take a try? SelectedFeatures are filtered from `workspaceConfigurableApps`, `workspaceConfigurableApps` will filter out default selected features.

Reply:
SelectedFeatures are filtered from workspaceConfigurableApps, workspaceConfigurableApps will filter out default selected features.